### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by msdrey**
- Added a readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,71 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Sewing Pattern Maker
 
-## Available Scripts
+Our web app allows users to generate a sewing pattern from a  photo of an article of clothing.
```
Comment (SarahPappas) [inline README.md]: This is silly, but I think there is an extra space in between "a" and "photo" "from a  photo"

Comment (SarahPappas) [inline README.md]: Maybe we should add the word basic, so it says "Our web app allows users to generate a basic sewing pattern..."

File: README.md [hunk 2]
```diff
@@ -1,23 +1,71 @@
...
 
-In the project directory, you can run:
+A user uploads a picture of a garment laying flat on a surface. Then, the user traces the seams, folds and borders of the garment using our integrated drawing tools, as well as providing a measurement. Finally, our app automatically generates a printable pdf pattern that will allow the user to sew a copy of the garment in the fabric of their choice!
```
Comment (SarahPappas) [inline README.md]: I think "Then, the user traces the seams, folds and borders of the garment using our integrated drawing tools, as well as providing a measurement." should be "Then, the user traces the seams, folds and borders of the garment using our integrated drawing tools. Next, they provide a measurement for a single seam of the original garment."

pdf should PDF

File: README.md [hunk 3]
```diff
@@ -37,8 +87,6 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## Built with
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (SarahPappas) [inline README.md:92]: Thanks for writing this up! It looks great!

File: README.md (hunk 1 above)
Comment (msdrey) [reply README.md]: good catch.

Comment (msdrey) [reply README.md]: I guess it would be good to slightly lower expectations, I agree.

File: README.md (hunk 3 above)
Comment (msdrey) [reply README.md:92]: Thanks!

File: README.md (hunk 2 above)
Comment (msdrey) [reply README.md]: I was trying to keep it short, but I agree it maybe wouldn't be clear to a first-time reader. I added your suggestion.

**Pull request merged**